src/ui/public/state_management/state_storage/hashed_item_store_singleton.js

```diff
@@ -0,0 +1,9 @@
+import HashedItemStore from './hashed_item_store';
+
+const instance = new HashedItemStore();
```

Comment:
I think this is overkill. Throughout the rest of kibana modules just export the "singleton" instance